### DISCUSSION TITLE
Windows XP Shutdown / Logoff Menu for Win10 x64

### DISCUSSION THREAD

**Discussion by ghost** [Open-Shell/Open-Shell-Menu#612] [Show and tell]
I hope this is in the right spot... I rebuilt (in AHK) the Luna-themed XP shutdown and logoff menus. They are quite close to the original and I used a lot of original components like buttons etc.
Here is a Youtube of it operational: [https://www.youtube.com/watch?v=bFKJyXArAJ4](url)
It is an awesome addition for Open-shell users who are on the XP Luna themes so I'm glad to be able to share it with you, it's still in beta but has support for other languages. Some things are not authentic though, like pressing CTRL to bring up the "shutdown with updates" feature we used to see in the XP days.
 You can make custom entries on the Open-shell "customize start menu" tab to point to these files and now it's much more authentic to the "before times" :) Here is the download page for you guys, I hope you enjoy it

[http://weedtrek.ca/XPshutdown.html](url)

Questions and comments can be directed at me personally using the email feature on the menu's download page, as I don't want to commandeer the fine github open-shell threads going on here. I'm also at the AutoHotkey forum as a member if you'd like to check my creds.
Aaron Bewza

![menu](https://user-images.githubusercontent.com/70411948/104794081-4159a800-575a-11eb-80d2-32bcfd5f8808.jpg)

**Comment by bonzibudd** [comment]
Cool, I am going to try this out! I will probably have some feature/suggestions if you are interested.

**Reply by ghost** [reply]
absolutely! I'd be honored :D

**Comment by TbBossHogg10172002** [comment]
How can you link this with open shell

**Reply by blackcrack** [reply]
hmm.. nice tool, but 1. is openshell not related to any program and not related to any version of shell surface,  
i loove XP too, and i think i make a plasmaicon-set of XP icons, but, OpenShell is build to be not related to any surfaces,  
therewith is your Proggy like Explorerpatcher  also as Tool same releated like other Programms.. Openshell should   
make with code extensions for run native in Win11 and 12, as make now more enhanced as tool,   
because this is a tool.. and OpenShell is not Windows versions releated.  
  
You could make a exefile out of your code to have a enhancement for put in in the System and stay to  OpenShell as Frend,  
btw, could you add in code a pluginsystem where it is possible for add some enhancement to have the normaluser for add  
some things and dll at the open shell, but specially a XP (Specific) Logout/restart/bootprogrogram hard coded in, is not the  
best idea for OpenShell, because it gives also Win311 Guy's or Windows7 and 10 Guy's who love to have an other Design  
and we whant not have a war between this surface, we wana' have a Happy party and modding possibilitys in OpenShell to  
have any of the worlds and the possibilibility to use different Surfaces in Skins of OpenShell, so, you have the possibility to  
add at OpenShell a Pluginsystem (what imho actually exist like i guess) and make like Rocketdock or Launchy a single Exe  
to put it as small WinNT changeout/add in Autostart or regestry autorun to start if need and toutch the actual system internal  
start or reboot automatism for start and let show the USers who whant need the users.....  
  
OpenShell use the System Internal algorithm like "shutdown.exe" with switches, make "shutdown /?" in the cmd or commandline of your Current WinNT system, i guess, shutdown exist since win2000 ? or WinXP and is the standard binary in WinNT in my opinion..

so would it be more nice if you help with (if you already in possibility to programming) for help and ask how we can make more opensource OpenShell as it is at actually in OpenShell, because this is the first step what should be fully finish to have a OpenSource Open Shell in Future for all users and not like an other Startmenu who is not open source and the user must food what the programmers closed source they gives to food..)

So, help with to make more open source in OpenShell code (what is need fast as possible at first) ,
and be in able to make self run enhancements (like you shutdowndialog) 
for modding different Win x86 and x86_64 WinNT Systems.. (include wine in Linux, wineboot ;) )


> How can you link this with open shell  

self running exe, make a link and put it on the desktop if it is not self running snap in in the WinNT System,
you can also put the link in Startmenu of your Useraccount as copy.. and if you have by standart on your desktop,
put it on you useraccout /startmenu/programs/autostart 
and have fun for often reboot and shutdown ;)

and by the way, why be the Orginal-Links change it out to an wrong place ?

best
Blacky

**Comment by ghost** [comment]
I link this (not directly) with Open-Shell because a lot of us use Open-Shell for the XP-style start menu... there are threads about this topic all over the place and all sorts of skins for it.
 But there is no XP shutdown menu in any of the skins, so I built this one to compliment the Open-Shell XP menu.
 No it is not part of the Open-Shell program, it is completely separate. I didn't quite finish wrapping it in an installer because AVG kept pinging it as a suspicious file so I spent a bunch of time fighting with them and really don't care to finish the program at this point, as it works the way it's supposed to.
 Not sure what the other commenter is saying but I understand one language (AHK) and I work in Windows only 🙂
 The site server must have been down when he checked the link.
________________________________

**Comment by Nick73** [comment]
Updated DL link? Also, what's the compatibility for Win11, if tested? Been looking for a Vista shutdown menu to complete my theme, but this will do for the meantime.

**Reply by ghost** [reply]
right here: https://github.com/blackwidowfarms/XPshutdown
- there is one anomaly with Windows 11, the "fade to grey" function suddenly uses too much contrast.
- there is no Vista anything, only XP
 Github wants to force me to use two-factor security to sign in , but I don't have a cell anymore so that is going to be impossible to sign in after the seven days they have given me.

**Reply by Nick73** [reply]
That links gives me 404, but I was able to find it on the WayBack Machine https://web.archive.org/web/20210516095833/http://weedtrek.ca/software/XP_Shutdown.zip

**Comment by erezyanay** [comment]
how do i link this to open shell?
